### ISSUE TITLE
How to limit the number of tags per repository

### ISSUE THREAD

**Issue by roldancer** [vmware/harbor#1431]
Hi, Is it possible to limit the number of tags per repository ? What I need is to define a quota policy by Project/repository/user, for example:

1) Define the total of repositories that a user can create.
2) Define the total of tags per repository a user can create.
3) Define the total of projects a user can create.

Thanks.

**Comment by cavemandaveman** [comment]
I agree, it would be nice to set a limit on the number of images in a repo/project and have the option to autodelete once that limit is reached.
